graphql/types/Product.graphql

```diff
@@ -261,9 +261,33 @@ type Offer {
   discountHighlights: [Discount!]
   teasers: [Teaser!]
   """
-  List of gifts associated with the product
+  List of SKUs for gifts associated with the product
   """
   giftSkuIds: [String]
+  """
+  List of gifts associated with the product
+  """
+  productGifts: [Gift]
+}
+
+type Gift {
+  productName: String
+  brand: String
+  linkText: String
+  description: String
+  productTitle: String
+  skuItem: GiftSkuItem
+}
+
+type GiftSkuItem {
+  nameComplete: String
+  images: [GiftImage]
+}
```

Comment:
Why not merge this into Gift type?

Reply:
No particular reason, guess those could be merged